peachjam/js/components/clipboard.ts

```diff
@@ -9,7 +9,7 @@ export class CopyToClipboard {
   }
 
   copy () {
-    navigator.clipboard.writeText(this.root.dataset.value || '')
+    navigator?.clipboard?.writeText(this.root.dataset.value || '')
```

Comment:
the `.then` will still fail. Instead, do `if (navigator.clipboard && navigator.clipboard.writeText)` or similar.